extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -123,6 +124,7 @@ public function getWikiVariables() {
 			'mainPageTitle' => Title::newMainPage()->getPrefixedDBkey(),
 			'theme' => SassUtil::getOasisSettings(),
 			'wikiCategories' => WikiFactoryHub::getInstance()->getWikiCategoryNames( $wgCityId ),
+			'enableNewAuth' => $wgEnableNewAuth
```

Comment:
Can you fix the alphabetic order? It was me that broke it in the first place :see_no_evil: